draft-ietf-httpbis-digest-headers.md

```diff
@@ -521,7 +524,7 @@ signatures.
 Using signatures to protect the checksum of an empty representation
 allows receiving endpoints to detect if an eventual payload has been stripped or added.
 
-Any mangling of digest fields, including de-duplication of representation-data-digest values
+Any mangling of digest fields, including de-duplication of `dict-member`s
 or combining different field values (see {{Section 5.2 of SEMANTICS}})
 might affect signature validation.
```

Comment:
Dictionaries already describe how to treat duplicate entries. So we probably need a separate issue to fix up this paragraph.

Reply:
Once a field is signed, even a compliant mangling invalidates the signature. I see this line as a simple advice to the users.